Core/GDCore/Events/Builtin/ForEachEvent.cpp

```diff
@@ -59,6 +71,17 @@ vector<const gd::Expression*> ForEachEvent::GetAllExpressions() const {
   return allExpressions;
 }
 
+vector<pair<const gd::Expression*, const gd::ParameterMetadata> >

```

Comment:
nitpicking: put the implementation of the const version immediately after the non const implementation

Reply:
Actually no, it's not how it was done in this file. Forget this! :)